lib/core/topologies/server.js

```diff
@@ -55,6 +55,10 @@ var serverAccounting = false;
 var servers = {};
 var BSON = retrieveBSON();
 
+function topologyId(server) {
+  return server.s.parent == null ? server.id : server.s.parent.id;
```

Comment:
is there ever a chance that `server.s` is null? Mu guess is no, but idk.

Reply:
nope